app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.find_by(email: params[:email])
+    
+    if user
+      user.generate_token!
+      AppMailer.send_forgot_password(user).deliver
+      redirect_to forgot_password_confirmation_path
+    else 
+      flash[:error] = params[:email].blank? ? "Email can not be blank." : 
```

Comment:
Oh wait, I just realized the next line is part of it....if you have to break the line that means it is too long to be a ternary. Use a traditional `if`/`else`.